src/tree/registries/auth/IAuthHelper.ts

```diff
@@ -0,0 +1,21 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as request from 'request-promise-native';
+import { URL } from 'url';
+import { ICachedRegistryProvider } from '../ICachedRegistryProvider';
+import { IDockerCliCredentials } from '../RegistryTreeItemBase';
+
+export interface IAuthHelper {

```

Comment:
```suggestion
export interface IAuthProvider {

```
'Helper' to me conveys the notion of a utility class with a few stateless methods, e.g. https://docs.microsoft.com/en-us/dotnet/api/microsoft.toolkit.uwp.helpers.packageversionhelper?view=win-comm-toolkit-dotnet-stable This is not the case here, hence `Provider`